pcsx2/R5900.cpp

```diff
@@ -435,11 +438,11 @@ __fi void _cpuEventTest_Shared()
 		cpuSetNextEventDelta( 48 );
 		//Console.Warning( "EE ahead of the IOP -- Rapid Event!  %d", EEsCycle );
 	}
-
+	
 	// The IOP could be running ahead/behind of us, so adjust the iop's next branch by its
 	// relative position to the EE (via EEsCycle)
-	cpuSetNextEventDelta( ((g_iopNextEventCycle-psxRegs.cycle)*8) - EEsCycle );
-
+	cpuSetNextEventDelta(((s32)((g_iopNextEventCycle - psxRegs.cycle)*psxEEmultiplier)) - EEsCycle);
+	
```

Comment:
Ditto

Reply:
Plus you've gone a bit overboard with the brackets